core/internal/features_test.go

```diff
@@ -345,6 +345,7 @@ func TestIntegration_ExternalAdapter_RunLogInitiated(t *testing.T) {
 	cltest.WaitForJobRunToPendIncomingConfirmations(t, app.Store, jr)
 
 	gethClient.On("BlockByNumber", mock.Anything, mock.Anything).Return(&types.Block{}, nil).Maybe()
+	newHeads := <-newHeadsCh
```

Comment:
Why?

Reply:
you get flakes if you let the Run in EthSubscribe return immediately instead of returning after app.Start()